src/client/pages/OfferNew/Introduction/HeroOfferDetails.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react'
+import styled from '@emotion/styled'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { OfferData } from 'pages/OfferNew/types'
+import { useTextKeys } from 'utils/textKeys'
+import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { getHouseholdSize } from '../utils'
+
+type Props = {
+  offerData: OfferData
+}
+
+const Container = styled.div`
+  width: 100%;
+  padding: 2rem 0;
+  color: ${colorsV3.white};
+
+  ${LARGE_SCREEN_MEDIA_QUERY} {
+    width: auto;
+    padding: 0;
+  }
+`
+
+const Header = styled.div`
```

Comment:
I agree it should probably be called `Heading` yes! 👍 

Regarding the semantics I made a decision to _not_ use semantic HTML since we don't have a strategy on that and in my mind it was better to just not use it at all until we actually have that. But maybe that goes against the overall style in the code base.